### PR TITLE
Update Chef Workstation supported platforms

### PR DESCRIPTION
**Pull request by vkarve-chef**
## Description
New OS versions in Ubuntu & RHEL are now supported. ARM architecture based systems are supported in macOS.

## Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

**Comment by vkarve-chef** [comment]
@IanMadd changes made in these pages
1. Install Workstation --> [Supported Platforms](https://deploy-preview-4111--chef-web-docs.netlify.app/workstation/install_workstation/#supported-platforms)
2. Overview --> [Supported Versions](https://deploy-preview-4111--chef-web-docs.netlify.app/versions/#supported-commercial-distributions)
3. Overview --> [Platforms](https://deploy-preview-4111--chef-web-docs.netlify.app/platforms/#chef-workstation)

File: _vendor/github.com/chef/chef-workstation/docs-chef-io/content/workstation/install_workstation.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Supported Host Operating Systems:
 </tr>
 <tr class="odd">
 <td>Red Hat Enterprise Linux / CentOS</td>
-<td>7.x, 8.x</td>
+<td>7.x, 8.x, 9.x</td>
```
Comment (IanMadd) [inline _vendor/github.com/chef/chef-workstation/docs-chef-io/content/workstation/install_workstation.md]: @vkarve-chef Can you undo this change. This should be made from the workstation repo. See chef/chef-workstation#3117

Comment (vkarve-chef) [reply _vendor/github.com/chef/chef-workstation/docs-chef-io/content/workstation/install_workstation.md]: Done ✅
